### PR TITLE
Added a button to star the repo

### LINKED ISSUE #12: Add 'Star this repo' button
Add a 'star this repo' button in the footer of the site.

### PR DESCRIPTION
**Pull request by Amarjit-Khan**
Added a 'Star this repo' button at the footer to 🌟 the repository.
Closes #12

### CODE REVIEW

**Review by mittal-parth — CHANGES_REQUESTED** [review]
@Amarjit-Khan Thanks for making a PR! However, seeing the output, I now feel that instead of 'Star this repo', it could be the GitHub icon followed by the word 'Star'.
The reason is the resume button should catch more attention than the Star repo button. 
You could add a screenshot before committing the changes.

**Comment by Amarjit-Khan** [comment]
Screenshot 2022-10-02 170419
What about this one?

**Comment by mittal-parth** [comment]
Looks great! You can push the changes

**Comment by Amarjit-Khan** [comment]
> Looks great! You can push the

**Comment by Amarjit-Khan** [comment]
> Looks great! You can push the changes
Ok Done.

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -4,6 +4,17 @@ import { socialMedia } from "../constants";
 import { profilePic } from "../assets";
 import { layout } from "../style";
 import { resumeLink } from "../constants";
+import { repoLink } from "../constants";
```
Comment (mittal-parth) [inline src/components/Footer.jsx:7]:
```suggestion
import { resumeLink, repoLink } from "../constants";
```

**Review by mittal-parth — CHANGES_REQUESTED** [review]
image

- [ ] The button isn't aligned in the same line. Can you look into that?
- [ ] Also, update your fork with the latest changes by following the [sample git workflow](https://github.com/mittal-parth/personal-portfolio/blob/main/readme.md#sample-git-workflow)

Thank you!

**Comment by Amarjit-Khan** [comment]
Screenshot 2022-10-02 194907
Made all suggested changes, corrected the allignment and fork is up to date.

**Review by mittal-parth — APPROVED** [review]

**Comment by mittal-parth** [comment]
Merged!
Thanks for the contribution! ✨

**Pull request merged**